PluginClasses/Zee5PluggablePlayer.swift

```diff
@@ -76,6 +82,42 @@ public class Zee5PluggablePlayer: APPlugablePlayerBase, ZPAdapterProtocol {
         
         instance.kalturaPlayerController = playerViewController
         
+        let Dict = items!.first!.extensionsDictionary
```

Comment:
I removed it from here in purpose, why it needed here? this whole init was moved into the playContent()

Reply:
Yes I checked That.